src/views/ScorePage/ScorePage.js

```diff
@@ -8,31 +8,50 @@ import Answer from '../../components/Answer/Answer';
 
 export function renderScorePage() {
   const recentUserScore = getCurrentScore(userAnswers);
-
+  const totalNumber = QuizSettings.questionsNum;
   const container = document.createElement('div');
   container.setAttribute('id', 'scorePage');
   container.append(
     renderGraphics(),
-    renderScore(recentUserScore),
+    renderScore(recentUserScore, totalNumber),
     renderNickForm(),
     renderMenuButton(),
     renderPlayAgainButton(),
   );
   document.querySelector('#app').append(container);
+  congratsMessage();
```

Comment:
recentUserScore and totalNumber are available in this scope, you can pass values those to congratsMessage function instead of creating new variables inside it